resources/js/features/home/components/+root/FrontPageNews/NewsCard.tsx

```diff
@@ -54,25 +64,51 @@ export const NewsCard: FC<NewsCardProps> = ({
           ) : null} */}
       </div>
 
-      <div>
-        <p className="mb-1 hidden text-2xs text-neutral-400/90 sm:block">
-          {PostedAt}{' '}
-          <span className="normal-case italic">
-            {'·'} {t('by {{authorDisplayName}}', { authorDisplayName })}
+      <div className="relative">
+        {news.pinnedAt && ziggy.device === 'desktop' ? (
+          <BaseTooltip>
+            <BaseTooltipTrigger className="absolute -right-2 -top-2">
+              <div className="flex size-8 items-center justify-center rounded-bl rounded-tr-lg bg-rose-600">
+                <LuPin className="-mt-1 size-5 text-white" />
```

Comment:
My OCD says this isn't vertically centered correctly.
![image](https://github.com/user-attachments/assets/d8877d07-448e-4650-9f41-e5dece0374f1)

Reply:
I'm having a lot of trouble getting this right - I think my brain is getting confused on centering within the box or centering within the box + the outline.

How does this look?
![Screenshot 2024-12-27 at 9 48 40 AM](https://github.com/user-attachments/assets/11bcef57-2ea0-4d15-bbbb-b699491ab928)